cmd/av/stack_next.go

```diff
@@ -50,16 +44,29 @@ var stackNextCmd = &cobra.Command{
 			return err
 		}
 
-		// confirm we can in fact do the operation given current branch state
-		if len(subsequentBranches) == 0 {
-			return errors.New("there is no next branch")
-		}
-		if n > len(subsequentBranches) {
-			return fmt.Errorf("invalid number (there are only %d subsequent branches in the stack)", len(subsequentBranches))
+		var branchToCheckout string
+		if !stackNextFlags.Last {
+			if len(subsequentBranches) == 0 {
+				return errors.New("there is no next branch")
+			}
+			n, err := strconv.Atoi(args[0])
+			if err != nil {
+				return errors.New("invalid number (unable to parse)")
+			}
+			if n <= 0 {
+				return errors.New("invalid number (must be >= 1)")
+			}
+			if n > len(subsequentBranches) {
+				return fmt.Errorf("invalid number (there are only %d subsequent branches in the stack)", len(subsequentBranches))
+			}
+			branchToCheckout = subsequentBranches[n-1]
+		} else {
+			if len(subsequentBranches) == 0 {
```

Comment:
Let's skip this check. Or at the very least, just print out a message and return `nil` here. I don't think it properly counts as an **error**.